pkg/test/framework/components/echo/echoboot/echoboot.go

```diff
@@ -0,0 +1,50 @@
+// Copyright 2019 Istio Authors
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package echoboot
```

Comment:
Why are we creating an echoboot package? Shouldn't this simple be echo/echo.go?

Reply:
The issue is that each implementation is separate files ... enough that warrants their own subpackages. The implementations need access to the top-level package structures.  So we can't have the creation code there ... so I moved it into a new package (echoboot).